editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/globalconfiguration/beans/validation/BeanValidationUtil.java

```diff
@@ -32,4 +34,12 @@ public static IStatus validateIdInParent(AbstractCamelModelElement parent, Objec
 		}
 		return ValidationStatus.ok();
 	}
+	
+	public static IStatus validateRequiredParemeter(Parameter parameter, Object value) {
+		if (PropertiesUtils.isRequired(parameter) && (value == null || value.toString().trim().length() < 1)) {
+			return ValidationStatus.error(UIMessages.propertyRequiredValidatorMandatoryParameterEmptyPt1 
```

Comment:
for internationalization, when it is in the same "message", it is better to use a single "externalized string" and use variables in .properties {0} and use NLS.bind